tailwind.config.cjs

```diff
@@ -62,6 +62,35 @@ module.exports = {
     },
     textColor: {
       ...colors,
+      secureonly: '#A98307',
+      domainmismatch: '#7D8471',
+      notonpath: '#79553D',
+      samesitestrict: '#AF2B1E',
+      samesitesax: '#FFA420',
+      samesiteunspecifiedtreatedaslax: '#FF7514',
+      samesitenoneinsecure: '#924E7D',
+      userpreferences: '#E63244',
+      thirdpartyphaseout: '#999950',
+      thirdpartyblockedinfirstpartyset: '#DE4C8A',
+      unknownerror: '#DE4C8A',
+      schemefulsamesitestrict: '#763C28',
+      schemefulsamesitelax: '#C6A664',
+      schemefulsamesiteunspecifiedtreatedaslax: '#E7EBDA',
+      samepartyfromcrosspartycontext: '#008F39',
+      namevaluepairexceedsmaxsize: '#1C542D',
+      excludesamesiteunspecifiedtreatedaslax: '#1C542D',
+      excludesamesitenoneinsecure: '#EDFF21',
+      excludesamesitelax: '#A5A5A5',
+      excludesamesitestrict: '#256D7B',
+      excludeinvalidsameparty: '#898176',
+      excludesamepartycrosspartycontext: '#497E76',
+      excludedomainnonascii: '#7FB5B5',
+      excludethirdpartycookieblockedinfirstpartyset: '#EFA94A',
+      excludethirdpartyphaseout: '#CF3476',
+      'blocked-frames': '#AF7AA3',
+      'cookie-frames': '#C5A06A',
+      'total-frames': '#25ACAD',
+      'unblocked-frames': '#F54021',
```

Comment:
Should these be color names?

Reply:
Used them how we are using classes for functional, marketing, analytics and uncategorised cookies. Making it consistent.

Reply:
Refactored the whole event handler and added some utility functions too for simplicity and better code readability.

Reply:
> Used them how we are using classes for functional, marketing, analytics and uncategorised cookies. Making it consistent.

Actually, that should also not be using such names. This list will keep growing, let's use color names and add a @todo comment for updating functional, marketing, analytics, uncategorized later.

Reply:
So instead of dynamically generating the color we will be keeping a map for reason and the color name if I am not wrong that is the approach we will be using?